enhancements/sig-architecture/81-addon-lifecycle/kustomize-rollout-examples/01-separately-placed-canaries/hub-scoped-addon-configuration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: HubScopedAddOnConfiguration
+metadata:
+  name: standard-config
```

Comment:
the reason this is cluster scoped is it is not per cluster configuration, but a global configuration. Is it correct understanding?

Reply:
> the reason this is cluster scoped is it is not per cluster configuration, but a global configuration. Is it correct understanding?

Correct.  The idea is that that some things like version are not usually subject the choices for individual spoke clusters and instead we'd like to manage them as group of spokes.